admin/acceptance.py

```diff
@@ -195,6 +203,23 @@ def ensure_keys(reactor):
             found.  Otherwise, fails with ``AgentNotFound`` or ``KeyNotFound``.
         """
 
+    def extend_cluster(reactor, cluster, count, tag, starting_index):
+        """
+        Create new nodes and add them to the given cluster.
+
+        :param reactor: Reactor to use.
```

Comment:
We are not super-consistent with documenting types. I always try to document them. But the reactor, as Jon explains above, is assumed to be self-evident.